packages/pancake-uikit/src/components/TabMenu/Tab.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import StyledTab from "./StyledTab";
+import { TabProps } from "./types";
+import { Text } from "../Text";
+
+const Tab: React.FC<TabProps> = ({ isActive = false, onClick, tabMenuLength = 1, children }) => {
+  const tabWidthPercentage = 100 / tabMenuLength;
```

Comment:
Why calculate the percentage and not use flex or grid?

Reply:
Also a good point - I was hung up on modifying the width style because it seemed to use a fixed width at desktop sizes in the designs, but looking again... it isn't.

Reply:
Nice suggestion. Used flex-grow instead. Just confirming the behaviour with Bugs is all good.

https://user-images.githubusercontent.com/79279477/111462088-ea4e6000-8715-11eb-87e4-1c9a56a8a5c8.mp4

Reply:
There were a few tiny padding tweaks but it has the Bugs 👍